modules/caddyhttp/fileserver/browse.html

```diff
@@ -799,15 +806,15 @@ <h1>
 						</span>
 						{{- end}}
 					</div>
-					<a href="javascript:queryParam('layout', '')" id="layout-list" class='layout{{if eq $.Layout "list" ""}}current{{end}}'>
+					<a href="#" id="layout-list" class='layout{{if eq $.Layout "list" ""}}current{{end}}'>
```

Comment:
One thing about `#` is that it changes the browser history, I believe. Can we replace this with `javascript:` without breaking CSP? `href="javascript:"`

Or at least use a ... what is it these days, preventDefault() or stopPropagation() or something? On the JS event handler, to prevent the actual page nav.

Reply:
I can remove the `href="#"` from both layout anchors and add a CSS rule to indicate that the element is clickable.